backend/application/core/models.py

```diff
@@ -520,6 +520,7 @@ class Observation(Model):
         on_delete=SET_NULL,
     )
     risk_acceptance_expiry_date = DateField(null=True)
+    purl_type = CharField(max_length=16, blank=True, null=True)
```

Comment:
This attribute shall have the name `origin_component_purl_type` and shall be placed directly behind `origin_component_purl`.

This name change shall be reflected in the other files using this attribute as well.

Reply:
Done